components/submit-grant/SubmitGrantFlow.tsx

```diff
@@ -163,28 +182,52 @@ function SubmitGrantDialog(props: SubmitGrantDialogProps) {
           </main>
           {submittedGrantId && (
             <DialogFooter className="z-30 flex justify-center border-t bg-[#BBE0FF] p-4 text-sm leading-5 text-[#184090] sm:justify-center sm:px-0">
-              <FormattedMessage
-                defaultMessage="<b>Grants are processed as 'expenses'.</b> To view your application, <link>View All Expenses</link>"
-                id="oAwTV8"
-                values={{
-                  b: I18nBold,
-                  link: c => {
-                    return (
-                      <React.Fragment>
-                        &nbsp;
-                        <Button
-                          className="h-5 px-0 leading-5"
-                          size="xs"
-                          variant="link"
-                          onClick={onViewAllExpensesClick}
-                        >
-                          {c}
-                        </Button>
-                      </React.Fragment>
-                    );
-                  },
-                }}
-              />
+              {hasGrantAndFundsReorgEnabled ? (
+                <FormattedMessage
+                  defaultMessage="To view your request,<link>View All Grants</link>"
+                  id="bPzQ2t"
+                  values={{
+                    link: c => {
+                      return (
+                        <React.Fragment>
+                          &nbsp;
+                          <Button
+                            className="h-5 px-0 leading-5"
+                            size="xs"
+                            variant="link"
+                            onClick={onViewAllGrantRequestsClick}
+                          >
+                            {c}
+                          </Button>
+                        </React.Fragment>
+                      );
+                    },
+                  }}
+                />
+              ) : (
+                <FormattedMessage
+                  defaultMessage="<b>Grants are processed as 'expenses'.</b> To view your application, <link>View All Expenses</link>"
+                  id="oAwTV8"
+                  values={{
+                    b: I18nBold,
+                    link: c => {
+                      return (
+                        <React.Fragment>
+                          &nbsp;
```

Comment:
There's a space before the <link> tag, it should be safe to remove this and React.Fragment.